### PR TITLE
Update ruleset: Eclipse.org

### PR DESCRIPTION
**Pull request by injust**

### CODE REVIEW

File: src/chrome/content/rules/Eclipse.xml [hunk 1]
```diff
@@ -1,26 +1,71 @@
...
-	<target host="www.eclipse.org" />
-	<target host="wiki.eclipse.org" />
+	<target host="help.eclipse.org" />
+	<target host="hudson.eclipse.org" />
+	<target host="iot.eclipse.org" />
+	<target host="kapua.eclipse.org" />
+	<target host="live.eclipse.org" />
+	<target host="lts.eclipse.org" />
 	<target host="marketplace.eclipse.org" />
-	<target host="eclipsecon.org" />
```
Comment (J0WI) [inline src/chrome/content/rules/Eclipse.xml]: Could you split this into a separate ruleset?

Comment (injust) [reply src/chrome/content/rules/Eclipse.xml]: Added in #15774. Thanks!

Comment (injust) [reply src/chrome/content/rules/Eclipse.xml]: Added, thanks!

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**